alpha-build-machinery/make/default.example.mk

```diff
@@ -19,13 +19,17 @@ CODEGEN_GROUPS_VERSION :=openshiftapiserver:v1alpha1
 # You can list all codegen related variables by:
 #   $ make -n --print-data-base | grep ^CODEGEN
 
+IMAGE_REGISTRY?=registry.svc.ci.openshift.org
+IMAGE_REPO?=$(IMAGE_REGISTRY)/ocp/4.3
```

Comment:
IMAGE_REPO doesn't contain the image registry part, same for IMAGE_TAG

I thing the correct nomenclature would be `IMAGE_REF :=$(IMAGE_REGISTRY)/$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)`

Reply:
or `IMAGE_REF:=$(IMAGE_REGISTRY)/$(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG)` where repository is `$(IMAGE_REGISTRY)/$(IMAGE_NAMESPACE)/$(IMAGE_NAME)` ? I am seeing conflicting sources. @sallyom wdyt?

Reply:
e.g. dockers claim the repository as the part before image name in 
```
To tag a local image with ID “0e5574283393” into the “fedora” repository with “version1.0”:

$ docker tag 0e5574283393 fedora/httpd:version1.0
```
https://docs.docker.com/engine/reference/commandline/tag/#tag-an-image-referenced-by-name-and-tag